server/models/getMaxPartySize.js

```diff
@@ -0,0 +1,15 @@
+const db = require('../db/index');
+
+module.exports = (req, res) => {
+  const { restaurantId } = req.params;
+
+  const queryStr = 'SELECT availableSeats FROM restaurants WHERE restaurantId = ? LIMIT 1';
+
+  return db.query(queryStr, [Number(restaurantId)], (err, data) => {
```

Comment:
Interesting that you choose to use callback-style here and promise-style in your front-end. Maybe in a future feature, refactor to use one style for code consistency?

Reply:
I thought mySql's query() returns a call back function instead of a promise object? Versus in the front end, I am using Axios, which returns a promise object.